CI_YAML.md

```diff
@@ -76,6 +76,49 @@ promote a test that has been passing for the past 50 runs.
 To prevent tests from rotting, all targets are required to have a clear owner. Add an
 owner in [TESTOWNERS](https://github.com/flutter/flutter/blob/master/TESTOWNERS)
 
+## Properties
+
+Targets support specifying properties that can be passed throughout infrastructure. The
+following are a list of keys that are reserved for special use. Note, properties is a
```

Comment:
nit. maybe make this bold or something, because it's probably the trickiest part of this whole setup and easy to miss when skimming this doc.